pkg/ccl/backupccl/backup_planning.go

```diff
@@ -65,6 +65,7 @@ const (
 	backupOptAsJSON          = "as_json"
 	backupOptWithDebugIDs    = "debug_ids"
 	backupOptIncStorage      = "incremental_location"
+	backupProgressDirectory  = "progress"
```

Comment:
lets move this with the other reserved directory and file names in manifest_handling. Lets also add a comment.